Main/src/main/java/com/main/acad/servlet/CookieFilter.java

```diff
@@ -0,0 +1,64 @@
+package com.main.acad.servlet;
+
+import com.main.acad.dao.SimpleUserDao;
+import com.main.acad.dao.UserDao;
+import com.main.acad.error.NoAccessToFileException;
+
+import java.io.IOException;
+import java.util.logging.Logger;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+
+public class CookieFilter implements Filter {
+    private static final Logger logger = Logger.getLogger(HttpServlet.class.getName());
+
+    @Override
+    public void init(FilterConfig filterConfig) throws ServletException {
+
+    }
+
+    @Override
+    public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) {
+        HttpServletRequest httpServletRequest = (HttpServletRequest) request;
+        String url = httpServletRequest.getRequestURL().toString();
+        Cookie arrayCookies[] = httpServletRequest.getCookies();
+        String cookieValue = "";
+        for (Cookie cokies : arrayCookies) {
+            cookieValue = cokies.getValue();
+        }
+        UserDao dao = SimpleUserDao.getInstance();
+        String userStatus = dao.findUserByLogin(cookieValue);
+        if (userStatus.equals("admin")) {
```

Comment:
Seriously? You check access by login? You should have in db additional column with role. When user logins, your login endpoint should identify user, make token from login, password, and registration date for example, id in db or something else using some encryption (Java contains base cryptographic package), then this token should be passed to cookie. Also it can set db id to cookie.
Every time user make request filter should retrieve by db id from cookie user, make the same encryption procedure as while login operation and compare new token with that from cookie, if check failed of cookie is empty we should redirect user to login or registration page. If no, we retrieve from valid user his role and decide whether he can access or not.